tools/sst_dump_test.cc

```diff
@@ -116,8 +120,24 @@ class SSTDumpToolTest : public testing::Test {
 
     // Populate slightly more than 1K keys
     uint32_t num_keys = kNumKey;
-    for (uint32_t i = 0; i < num_keys; i++) {
-      tb->Add(MakeKey(i), MakeValue(i));
+    const std::string comparator_name = ikc.user_comparator()->Name();
+    if (comparator_name == ReverseBytewiseComparator()->Name()) {
```

Comment:
Nit: very minor. there are several implicit data cast and string creation here. It may be easier to just use `strcmp` for the two comparator's `Name()`. Not a big deal for test.